devtools/default_dev_container

```diff
@@ -1 +1 @@
-rsyslog/rsyslog_dev_base_ubuntu:16.04
+rsyslog/rsyslog_dev_base_ubuntu:19.10
```

Comment:
I would be a bit careful here - 19.10 is an interim release with EOL in around 3 months. Isn't 18.04 sufficient? Else I can see that we get 20.04 in place.

Reply:
@rgerhards just got it from rsyslog 
https://github.com/rsyslog/rsyslog/blob/master/devtools/default_dev_container

Reply:
oops ;-) some more thing to do... OK, I'll look at it.

Reply:
So we leave it for 19:10 for now? At least we can use latest clang here which already found a new race while making this PR here.